docs/docs/recipes/customize-sie/configure-sign-in-methods.mdx

```diff
@@ -2,68 +2,95 @@ import SaveChanges from './fragments/_save-changes.md';
 
 # Configure sign-in methods
 
-:::note
-Although the term only has "sign-in", to simplify, all these methods also imply the "sign-up method" (if applicable).
-:::
-
 <SaveChanges />
 
-## Methods
+Here is a list of the flows and UI elements we supported up to our Sign-in Experience 2.0. Based on those elements, you can use Logto straight out of the box or modify it to suit your own company requirements.
+
+![Sign-in Experience 2.0 Flows](./assets/sie_methods.png)
```

Comment:
this image looks confusing to me, i'll talk to designers

Reply:
discussed with designers, line 7-9 can be removed for now.

Reply:
updated